modules/distribution/src/main/assembly/bin.xml

```diff
@@ -281,6 +281,21 @@
             <destName>axis2.xml</destName>
         </file>
         <file>
+            <source>src/main/conf/web.xml</source>
+            <outputDirectory>wso2esb-${pom.version}/repository/conf/tomcat</outputDirectory>
+            <destName>web.xml</destName>
+        </file>
+        <file>
+            <source>src/main/conf/carbon/web.xml</source>
+            <outputDirectory>wso2esb-${pom.version}/repository/conf/tomcat/carbon/WEB-INF</outputDirectory>
+            <destName>web.xml</destName>
+        </file>
+        <file>
+            <source>src/main/conf/webapp-mode/web.xml</source>
+            <outputDirectory>wso2esb-${pom.version}/webapp-mode/WEB-INF</outputDirectory>
```

Comment:
AFAIK, webapp-mode is no longer used. This was used to deploy Carbon as a webapp in other application servers.